ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -2901,6 +2901,14 @@ static int zend_jit_setup(void)
 		asm ("movq _tsrm_ls_cache@gottpoff(%%rip),%0"
 			: "=r" (ret));
 		tsrm_ls_cache_tcb_offset = ret;
+#elif defined(__MUSL__)
```

Comment:
is there any simple way this can be tested?

Reply:
No simple way, but I asked Shivam about setting up a build with Alpine/musl. He'll give it a shot.